visualizers/visualization-common.css

```diff
@@ -0,0 +1,122 @@
+body {
+    margin:      0;
+    padding:     0;
+    font-size:   12px;
+    line-height: 16px;
+}
+
+/* custom scrollbars */
+
+html::-webkit-scrollbar {
+    width:  4px;
+    height: 4px;
+    background: transparent;
+}
+html::-webkit-scrollbar-thumb {
+    background: rgba(255,255,255,0);
+    border-radius: 5px;
+}
+html:hover::-webkit-scrollbar-thumb {
+    background: rgba(255,255,255,0.05);
+}
+
+/* hasklig */
+
+@font-face {
+    font-family: 'hasklig-extra-light';
+    src: url('hasklig/extra-light.otf');
+    }
+    @font-face {
```

Comment:
that indentation is confusing (the second `@font-face` shouldn't be indented at all.